packages/material/scss/tooltip/_theme.scss

```diff
@@ -0,0 +1,11 @@
+$tooltip-color: $base-bg !default;
+$tooltip-bg: rgba(lighten($base-text, 38), .9) !default;
+
+@import "~@progress/kendo-theme-default/scss/tooltip/theme";
+
+@include exports("tooltip/theme/material") {
+
+    .k-callout {
+        border-color: transparent;
```

Comment:
If the idea is to have no callout, why not hide it e.g.:

```scss
// in _layout.scss, right after the import

@include exports("tooltip/layout/material") {
    .k-callout { display: none; }
}
```

Another way, would be to have a flag for that. @gyoshev what's your thought on the flag?

Reply:
I don't think it is wise to always hide the callout. Both the jQuery and Angular tooltips have a `callout` configuration option, and adding either rules will disable the effect that the property has, even if customers use `callout: true`, which is frustrating.